guides/search.md

```diff
@@ -0,0 +1,289 @@
+# Get the last sort value from the first page
+search_after = hits[-1]['sort']
+# Fetch page 2
+search_body["search_after"] = search_after
+response = client.search(
+    index='movies',
+    body=search_body
+)
+hits_page_2 = response['hits']['hits']
+# Get the last sort value from page 2
+search_after = hits_page_2[-1]['sort']
+# Fetch page 3
+search_body["search_after"] = search_after
+response = client.search(
+    index='movies',
+    body=search_body
+)
+hits_page_3 = response['hits']['hits']
+# Print the hits from each page
+print("Page 1:")
+for hit in hits:
+    print(hit)
+print("\nPage 2:")
+for hit in hits_page_2:
+    print(hit)
+print("\nPage 3:")
+for hit in hits_page_3:
+    print(hit)
+```
+
+
+### Pagination with scroll
+
+When retrieving large amounts of non-real-time data, you can use the `scroll` parameter to paginate through the search results. 
+
+```python
+from opensearchpy import OpenSearch
+# Create an OpenSearch client with appropriate hosts and connection details
+client = OpenSearch(hosts=['localhost'])
+# Define the search query with scroll and pagination options
+search_body = {
+    "query": {
+        "match": {
+            "title": "dark knight"
+        }
+    },
+    "scroll": "1m",  # Set the scroll duration to 1 minute
+    "size": 2
+}
+# Perform the initial search operation on the 'movies' index with the defined query and scroll options
+page_1 = client.search(
+    index='movies',
+    body=search_body
+)
+# Extract the scroll_id from the response
+scroll_id = page_1['_scroll_id']
+# Perform the scroll operation to get the next page of results
+page_2 = client.scroll(
+    scroll_id=scroll_id,
+    scroll='1m'
+)
+# Extract the scroll_id from the response
+scroll_id = page_2['_scroll_id']
+# Perform another scroll operation to get the third page of results
+page_3 = client.scroll(
+    scroll_id=scroll_id,
+    scroll='1m'
+)
+# Extract the hits from each page of results
+hits_page_1 = page_1['hits']['hits']
+hits_page_2 = page_2['hits']['hits']
+hits_page_3 = page_3['hits']['hits']
+```
+
+
+
+### Pagination with Point in Time
+
+The scroll example above has one weakness: if the index is updated while you are scrolling through the results, they will be paginated inconsistently. To avoid this, you should use the "Point in Time" feature. The following example demonstrates how to use the `point_in_time` and `pit_id` parameters to paginate through the search results:
+
+```python
+from opensearchpy import OpenSearch
+# Create an OpenSearch client with appropriate hosts and connection details
+client = OpenSearch(hosts=['localhost'])
+# Define the search query with sorting and pagination options
+search_body = {
+    "query": {
+        "match": {
+            "title": "dark knight"
+        }
+    },
+    "sort": [
+        {
+            "year": {
+                "order": "asc"
+            }
+        }
+    ]
+}
+# Create a point in time
+pit = client.create_pit(index='movies', keep_alive='1m')
```

Comment:
The API is `create_point_in_time`. See https://github.com/opensearch-project/opensearch-py/blob/main/USER_GUIDE.md#point-in-time-api.